man/fi_fabric.3.md

```diff
@@ -114,6 +114,10 @@ datatype or field value.
 *FI_TYPE_MSG_ORDER*
 : struct fi_ep_attr::msg_order field
 
+*FI_TYPE_VERSION*
+: Returns the library version of libfabric in string form.  The data
+parameter is ignored.
+
```

Comment:
This is *super* nit-picky...

I actually don't know/remember offhand if this matters, but all the other man pages are of the form:

```
*FOO*
: First line of the text is just like you'd expect.
  The second line is (and all subsequent lines are) indented 2 spaces.
```

I could easily be wrong here, but I have a (dim) recollection that the 2-space indenting makes a difference somehow.

Regardless (and this is the *super* picky part...) it would probably be good to put it in the same form as the rest of the text, anyway.

Thanks for coping with my type-A personality OCD.  :-)